src/client/eppo-client.ts

```diff
@@ -72,42 +44,30 @@ export interface IEppoClient {
   getStringAssignment(
     subjectKey: string,
     flagKey: string,
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    subjectAttributes?: Record<string, any>,
-    assignmentHooks?: IAssignmentHooks,
-  ): string | null;
+    defaultValue: string,
+    subjectAttributes?: Record<string, AttributeType>,
+  ): string;
```

Comment:
Given that there are a bunch of breaking changes already, do we want to update the signature here.

I think I've heard @aarsilv suggest changing this to:

`getXAssignment(flagKey, defaultValue, subjectKey, subjectAttributes?)` to keep subjectAttributes closer to subjectKey

Thoughts @aarsilv @giorgiomartini0?

Reply:
I slightly prefer the existing signature; the flow feels more natural. ("get String assignment of `<subjectKey>` for `<flagKey>` with default `<defaultValue>`", vs "For flag `<flagKey>`, with a default of `<defaultValue>`, get string assignment of `<subjectKey>`" – this feels jumbled to me.)

Reply:
My opinion is a loosely held one, I'm ok with the existing one as well. Or maybe poll in the `#engineering` Slack Channel 😓